building source on Mac has one error... but succeeds.

Here's the output of `Invoke-Build`:

```
Invoke-Build
Downloading dotnet version 2.1.2
Build . /Users/tylerleonhardt/Desktop/CompSci/DotNET/UnixConsoleEcho/UnixConsoleEcho.build.ps1
Task /./Build/Clean

Welcome to .NET Core!
---------------------
Learn more about .NET Core @ https://aka.ms/dotnet-docs. Use dotnet --help to see available commands or go to https://aka.ms/dotnet-cli-docs.

Telemetry
--------------
The .NET Core tools collect usage data in order to improve your experience. The data is anonymous and does not include command-line arguments. The data is collected by Microsoft and shared with the community.
You can opt out of telemetry by setting a DOTNET_CLI_TELEMETRY_OPTOUT environment variable to 1 using your favorite shell.
You can read more about .NET Core tools telemetry @ https://aka.ms/dotnet-cli-telemetry.

Configuring...
-------------------
A command is running to initially populate your local package cache, to improve restore speed and enable offline access. This command will take up to a minute to complete and will only happen once.
Decompressing 100% 5816 ms
Expanding 100% 12243 ms
Microsoft (R) Build Engine version 15.5.179.9764 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

Done /./Build/Clean 00:00:21.4399750
Task /./Build/BuildNative
/Users/tylerleonhardt/Desktop/CompSci/DotNET/UnixConsoleEcho/src/Native/Unix/disable_key_echo.cpp:20:23: error: use of undeclared identifier 'nullptr'
    assert(termios != nullptr);
                      ^
1 error generated.
Done /./Build/BuildNative 00:00:01.9985450
Task /./Build/BuildDll
Microsoft (R) Build Engine version 15.5.179.9764 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  Restoring packages for /Users/tylerleonhardt/Desktop/CompSci/DotNET/UnixConsoleEcho/src/UnixConsoleEcho/UnixConsoleEcho.csproj...
  Generating MSBuild file /Users/tylerleonhardt/Desktop/CompSci/DotNET/UnixConsoleEcho/src/UnixConsoleEcho/obj/UnixConsoleEcho.csproj.nuget.g.props.
  Generating MSBuild file /Users/tylerleonhardt/Desktop/CompSci/DotNET/UnixConsoleEcho/src/UnixConsoleEcho/obj/UnixConsoleEcho.csproj.nuget.g.targets.
  Restore completed in 520.81 ms for /Users/tylerleonhardt/Desktop/CompSci/DotNET/UnixConsoleEcho/src/UnixConsoleEcho/UnixConsoleEcho.csproj.
  UnixConsoleEcho -> /Users/tylerleonhardt/Desktop/CompSci/DotNET/UnixConsoleEcho/src/UnixConsoleEcho/bin/Debug/netstandard1.1/UnixConsoleEcho.dll

Build succeeded.
    0 Warning(s)
    0 Error(s)

Time Elapsed 00:00:01.88
Done /./Build/BuildDll 00:00:04.1374220
Done /./Build 00:00:27.6018100
Done /. 00:00:27.6104170
Build succeeded. 5 tasks, 0 errors, 0 warnings 00:00:49.9981540
```

You can see the error here:
```
/Users/tylerleonhardt/Desktop/CompSci/DotNET/UnixConsoleEcho/src/Native/Unix/disable_key_echo.cpp:20:23: error: use of undeclared identifier 'nullptr'
    assert(termios != nullptr);
                      ^
1 error generated.
```

This doesn't look ok, but maybe it is?

--- a/UnixConsoleEcho.build.ps1
+++ b/UnixConsoleEcho.build.ps1
@@ -41,7 +41,7 @@ task BuildNative {
     }
 
     # If not Windows or Linux then assume MacOS
-    g++ -dynamiclib -o $PSScriptRoot/src/Native/Unix/build/libdisablekeyecho.dylib $PSScriptRoot/src/Native/Unix/disable_key_echo.cpp
+    g++ -dynamiclib -o $PSScriptRoot/src/Native/Unix/build/libdisablekeyecho.dylib $PSScriptRoot/src/Native/Unix/disable_key_echo.cpp -std=c++0x
 }
 
 task DoPack {

